tests/python/gaia-ui-tests/gaiatest/apps/clock/regions/new_alarm.py

```diff
@@ -0,0 +1,78 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.clock.app import Clock
+
+
+class NewAlarm(Clock):
+
+    _alarm_name_locator = (By.ID, 'alarm-name')
+    _alarm_view_locator = (By.CSS_SELECTOR, 'section.skin-dark.panel.active')
```

Comment:
I'm not sure this is a great locator to use. It seems rather general (and theme specific?). Perhaps it's better to use section[data-panel-id="alarm_edit"] here?